src/app/devices/views/DeviceDetails/DeviceNetwork/DeviceNetwork.test.tsx

```diff
@@ -44,19 +37,12 @@ describe("DeviceNetwork", () => {
       }),
     });
     const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/device/abc123", key: "testKey" }]}
-        >
-          <CompatRouter>
-            <DeviceNetwork systemId="abc123" />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
-    );
-    expect(wrapper.find("NodeNetworkTab").exists()).toBe(true);
-    expect(wrapper.find("DHCPTable").exists()).toBe(true);
-    expect(wrapper.find("Spinner").exists()).toBe(false);
+    renderWithBrowserRouter(<DeviceNetwork systemId="abc123" />, {
+      route: "/device/abc123",
+      store,
+    });
+    expect(screen.getByLabelText("Device network")).toBeInTheDocument();
+    expect(screen.getAllByRole("grid")).toHaveLength(2);
```

Comment:
Without looking at the implementation - I'm assuming the length will be 1 when a specific name is used? 

```suggestion
    expect(screen.getAllByRole("grid", { name: /DHCP/ )).toBeInTheDocument();
```

Reply:
That works actually yeah! Only thing is the first table doesn't appear to have a name, so I just gave it an aria label